test/helper.js

```diff
@@ -1,10 +1,14 @@
 // test globals
+/* global assert,debug,fs,path,util,exec */
+/* eslint-disable */
```

Comment:
Yes, because eslint doesn't see a `var debug ..` anywhere, it later complains on use of debug, even though it doesn't complain on these require lines.